### ISSUE TITLE
API: Collecting contact information after API registration

### ISSUE THREAD

**Issue by emmambd** [MobilityData/mobility-feed-api#99]
### Describe the problem

We need to store the contact data collected in the form that users see [after they register for an API account](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=143%3A57&mode=design&t=7rbLazl6G1H48QOD-1). 

### Proposed solution

Tasks:

 Setup Firestore. Check cost.
 Implement save info to Firestore.

**Comment by davidgamez** [comment]
Tasks:
- [ ] Setup Firestore. Check cost.
- [ ] Implement save info to Firestore.
- [ ] Implement logic to show the screen after sign-in.

**Comment by davidgamez** [comment]
Blocked by #118

**Comment by emmambd** [comment]
Duplicate of #113
